src/sage/rings/function_field/differential.py

```diff
@@ -190,16 +194,17 @@ def _richcmp_(self, other, op):
 
         EXAMPLES::
 
-            sage: K.<x> = FunctionField(GF(4)); _.<Y> = K[]                             # optional - sage.rings.finite_rings
-            sage: L.<y> = K.extension(Y^3 + x + x^3*Y)                                  # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: w1 = y.differential()                                                 # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: w2 = L(x).differential()                                              # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: w3 = (x*y).differential()                                             # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: w1 < w2                                                               # optional - sage.rings.finite_rings sage.rings.function_field
+            sage: # needs sage.rings.finite_rings
+            sage: K.<x> = FunctionField(GF(4)); _.<Y> = K[]
+            sage: L.<y> = K.extension(Y^3 + x + x^3*Y)                                  # needs sage.rings.function_field
```

Comment:
and why not adding `sage.rings.function_field` to the block like `sage: # needs sage.rings.finite_rings sage.rings.function_field`

Reply:
Done, thanks